okta/resource_okta_user_test.go

```diff
@@ -237,6 +237,45 @@ func TestAccOktaUser_updateAllAttributes(t *testing.T) {
 	})
 }
 
+func TestAccOktaUser_updateCredentials(t *testing.T) {
```

Comment:
@quantumew - Have added a separate test for credentials update so that other setup files remain intact. Let me know if this works. I have tested it and it's passing.

Reply:
Awesome work! Going to review this shortly. Running tests on #341. Might be just a bit before I get to the acc tests.